README.md

```diff
@@ -66,20 +77,116 @@ sentry::path: '/var/lib/sentry'
 sentry::version: '7.7.1'
 ```
 
+## Classes
```

Comment:
The Puppetlabs modules (usually) enumerate all of the class and defined type parameters.  I agree it adds more work to us to maintain this module, but it makes referencing parameters much easier: one need only look at README, rather than look at each of the manifest files individually.

I propose we leave these in place for now, and if it becomes too onerous to keep up-to-date we can remove them in the future.